### PR TITLE
add conditional csp flag

### PR DESCRIPTION
**Pull request by pgvishnuram**
## Description

This PR allows external src to allow conditionally with astronomer platform  

## Related Issues

- https://github.com/astronomer/issues/issues/6749
- https://github.com/astronomer/issues/issues/6709

## Testing

QA should able to open the playground without any errors in webbrowser console logs by default

## Merging

cherry-pick to release-0.36

### CODE REVIEW

File: charts/nginx/values.yaml [hunk 1]
```diff
@@ -124,3 +124,10 @@ defaultBackend:
   resources: {}
   readinessProbe: {}
   livenessProbe: {}
+
+cspPolicy:
```
Comment (rishkarajgi) [inline charts/nginx/values.yaml:128]: cdnPolicy

Comment (rishkarajgi) [inline charts/nginx/values.yaml:128]:
```suggestion
cdnPolicy:
```

Comment (pgvishnuram) [reply charts/nginx/values.yaml:128]: this change comes under csp policy

Comment (rishkarajgi) [reply charts/nginx/values.yaml:128]: then let's change `enabled` to `cdnEnabled`

Comment (rishkarajgi) [reply charts/nginx/values.yaml:128]: `cspPolicy.cdnEnabled` looks more legible

File: charts/nginx/values.yaml [hunk 2]
```diff
@@ -124,3 +124,9 @@ defaultBackend:
...
+  enabled: true
```
Comment (rishkarajgi) [inline charts/nginx/values.yaml:129]:
```suggestion
  cdnEnabled: true
```

**Review by rishkarajgi — APPROVED** [review]

**Pull request merged**